src/LogNormal/LogNormalMath.sol

```diff
@@ -80,12 +81,10 @@ function computeYGivenL(
     uint256 S,
     LogNormalParams memory params
 ) pure returns (uint256 ry) {
-    int256 d2 = computeD2(S, params);
-    int256 cdf = Gaussian.cdf(d2);
-    uint256 unsignedCdf = toUint(cdf);
+    int256 d2 = computeD2({ S: S, params: params });
```

Comment:
Why are you using named parameters here, just curious?

Reply:
we should always used named parameters because most parameters use the same type (uint) its easy to put them in the wrong order